compute_tools/src/bin/fast_import.rs

```diff
@@ -180,10 +181,10 @@ impl PostgresProcess {
             .context("create pgdata directory")?;
 
         let pg_version = match get_pg_version(self.pgbin.as_ref()) {
-            PostgresMajorVersion::V14 => 14,
-            PostgresMajorVersion::V15 => 15,
-            PostgresMajorVersion::V16 => 16,
-            PostgresMajorVersion::V17 => 17,
+            PostgresMajorVersion::V14 => PgMajorVersion::PG14,
+            PostgresMajorVersion::V15 => PgMajorVersion::PG15,
+            PostgresMajorVersion::V16 => PgMajorVersion::PG16,
+            PostgresMajorVersion::V17 => PgMajorVersion::PG17,
```

Comment:
Can't this just be a `PgMajorVersion::from(PostgresMajorVersion)`?

Reply:
`From<PostgresMajorVersion>` is not implemented for `PgMajorVersion`, among others because it's out of scope for the `postgres_versioninfo` library to implement casts form/to users' own types.

So far, there's only one user which wants to change PostgresMajorVersion into PgMajorVersion, and that's here. A future modification may want to add that cast, but I'd rather keep that outside the scope of this PR.

Reply:
Usage of `PostgresMajorVersion` has been replaced with `PgMajorVersion`, `PostgresMajorVersion` has been dropped.